docs/server/project_setup.rst

```diff
@@ -116,7 +116,25 @@ unless you need to further customize your project.
 
 
 Once you are done click on the **Save** button below the form to create the
-project. Creating the project doesn't actually import all the translations to
+project.
+
+Please not that **just creating the project is not enough**, now it is
+necessary to set the the filesystem configuration. So now click on the
+``Filesystems`` link below the project edit form and set the following:
+
+.. image:: ../_static/pootle_fs_link.png
+
+
+* **Filesystem backend** to ``localfs``
+* **Backend URL or path** to the value of
+  :setting:`POOTLE_TRANSLATION_DIRECTORY` + project code, e.g.
```

Comment:
This is not true, you can set it to anything accessible, it is not related to `POOTLE_TRANSLATION_DIRECTORY` at all.

Reply:
it kinda is in that this is the default directory

Reply:
Since this is an example I tried to keep it as simple and familiar as possible. Note the text doesn't tell in any way that it can't have any other value here.